{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "97d558b6_25424209",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1380460
      },
      "writtenOn": "2021-03-22T22:48:50Z",
      "side": 1,
      "message": "Adding Paul as reviewer as the author of the original error throwing logic.\n@aerotwist: Hi Paul, PTAL, thanks!",
      "revId": "45d1382d277c519a587fa2ef79dce3bc2f23864c",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}